ui/coordinateTrainer/src/chessground.ts

```diff
@@ -0,0 +1,38 @@
+import { h, VNode } from 'snabbdom';
+import { Chessground } from 'chessground';
+import { Config as CgConfig } from 'chessground/config';
+import * as cg from 'chessground/types';
+import resizeHandle from 'common/resize';
+import CoordinateTrainerCtrl from './ctrl';
+
+export default function (ctrl: CoordinateTrainerCtrl): VNode {
+  return h('div.cg-wrap', {
+    hook: {
+      insert: vnode => {
+        const el = vnode.elm as HTMLElement;
+        ctrl.chessground = Chessground(el, makeConfig(ctrl));
+      },
+      destroy: () => ctrl.chessground!.destroy(),
+    },
+  });
+}
+
+function makeConfig(ctrl: CoordinateTrainerCtrl): CgConfig {
+  //TODO prevent board from being highlight able
```

Comment:
One thing I couldn't figure out - what is the config setting for disallowing the selection of squares like this?
![image](https://user-images.githubusercontent.com/12655228/157113721-c5a84cb1-4ee0-4226-bda8-a2df7cb5911e.png)

Reply:
Fixed it in what might be a hacky way in bfa98d3

Reply:
Setting `movable.color` to `null` or `undefined` shuld do it.

Reply:
🎉  that did it!